common/src/main/java/org/mcaccess/minecraftaccess/mixin/AbstractContainerScreenAccessor.java

```diff
@@ -0,0 +1,24 @@
+package org.mcaccess.minecraftaccess.mixin;
+
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(AbstractContainerScreen.class)
+public interface AbstractContainerScreenAccessor {
+    @Accessor
+    int getLeftPos();
+
+    @Accessor
+    int getTopPos();
+
+    @Accessor
+    AbstractContainerMenu getMenu();
+
+    @Accessor
+    int getImageWidth();
+
+    @Accessor
+    int getImageHeight();
+}
```

Comment:
No newline at end of file.
```suggestion
}

```

Reply:
When I try accepting this it says:
Suggestion cannot be identical to original text.